### PR TITLE
[14.0][ADD] website_forum_subscription: New module

### PR DESCRIPTION
**Pull request by chienandalu**
Allows that users can subscribe or unsubscribe from forum notifications.

image

cc @Tecnativa TT39738

please review @CarlosRoca13 @LudLaf

### CODE REVIEW

File: website_forum_subscription/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Allows that users can subscribe or unsubscribe from forum notifications.
```
Comment (LudLaf) [inline website_forum_subscription/readme/DESCRIPTION.rst]: Allows users to subscribe/unsubscribe from forum notifications.

File: website_forum_subscription/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+To use this module, you need to:
+
+#. Go to your website /forum page.
```
Comment (LudLaf) [inline website_forum_subscription/readme/USAGE.rst]: #. Go to your website/forum page.

File: website_forum_subscription/readme/USAGE.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+#. If there are several forums, you'll find a subscription button in every card. Click
+   on it to subscribe or unsubscribe depending on the state of your forum subscription.
```
Comment (LudLaf) [inline website_forum_subscription/readme/USAGE.rst]: on it to subscribe or unsubscribe depending on your forum subscription state.

File: website_forum_subscription/readme/USAGE.rst [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+#. In the main forum page, you'll find the same button on the top of the forum panel.
```
Comment (LudLaf) [inline website_forum_subscription/readme/USAGE.rst]: #. In the main forum page, you'll find the same button on top of the forum panel.

**Review by LudLaf — CHANGES_REQUESTED** [review]
Please check "DESCRIPTION.rst" and "USAGE.rst" minor changes.

**Comment by chienandalu** [comment]
Fixed, @LudLaf

**Review by CarlosRoca13 — APPROVED** [review]

**Review by wytzejan — APPROVED** [review]

**Pull request merged**